### PR TITLE
Fixing the common view check condition.

### PR DESCRIPTION
**Pull request by zhljen**

### CODE REVIEW

File: metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/sql/DirectSqlTable.java [hunk 1]
```diff
@@ -263,8 +263,7 @@ private void validateTableType(final QualifiedName tableName, final Map<String,
             if (ICEBERG_TABLE_TYPE.equalsIgnoreCase(tableMetadata.get(PARAM_TABLE_TYPE))) {
                 return;
             }
-            if (VIRTUAL_VIEW_TABLE_TYPE.equalsIgnoreCase(tableMetadata.get(PARAM_TABLE_TYPE))
-                && tableMetadata.containsKey(COMMON_VIEW)
+            if (tableMetadata.containsKey(COMMON_VIEW)
```
Comment (ajoymajumdar) [inline metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/sql/DirectSqlTable.java]: The two condition check can be replaced with one. Boolean.parseBoolean(tableMetadata.get(COMMON_VIEW))

Comment (zhljen) [reply metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/sql/DirectSqlTable.java]: Changed based on comments with a bit code refactor.

File: metacat-connector-hive/src/test/groovy/com/netflix/metacat/connector/hive/HiveConnectorUtilSpec.groovy [hunk 2]
```diff
@@ -42,4 +45,19 @@ class HiveConnectorUtilSpec extends Specification{
         '1'         | 1L
         '010m'      | 600L
     }
+
+    def "Test for check common view" () {
+        given:
+        def tableInfo = TableInfo.builder().metadata(metadata).build()
+        def isView = HiveTableUtil.isCommonView(tableInfo)
+        def isView2 = HiveTableUtil.isCommonView(tableInfo)
```
Comment (danielbwatson) [inline metacat-connector-hive/src/test/groovy/com/netflix/metacat/connector/hive/HiveConnectorUtilSpec.groovy]: I think isView2 can be removed.

File: metacat-connector-hive/src/test/groovy/com/netflix/metacat/connector/hive/HiveConnectorUtilSpec.groovy [hunk 3]
```diff
@@ -42,4 +45,19 @@ class HiveConnectorUtilSpec extends Specification{
...
+        expect:
+        isView == output
+        isView2 == output
+        where:
+        metadata     | output
+        ['common_view' : "true"]    | true
+        ['common_view' : "false"]    | false
+        ['common_view_1' : "false"]    | false
```
Comment (danielbwatson) [inline metacat-connector-hive/src/test/groovy/com/netflix/metacat/connector/hive/HiveConnectorUtilSpec.groovy]: I'd like to add checks for the metadata `[:]`.  Also if metadata is nullable (I haven't checked) we should add the `null` case here as well.

Comment (zhljen) [reply metacat-connector-hive/src/test/groovy/com/netflix/metacat/connector/hive/HiveConnectorUtilSpec.groovy]: Added more cases.

File: metacat-connector-hive/src/test/groovy/com/netflix/metacat/connector/hive/HiveConnectorUtilSpec.groovy (hunk 2 above)
Comment (zhljen) [reply metacat-connector-hive/src/test/groovy/com/netflix/metacat/connector/hive/HiveConnectorUtilSpec.groovy]: I actually didn't notice this and it's supposed to test a different function. I updated the test and function code. I will merge the code after passing the build.

**Pull request merged**